app/views/notifier_mailer/pending_poll_email.html.erb

```diff
@@ -21,11 +21,11 @@
     </p>
   <% end %>
 
-  <% if @poll.affected_user %>
+  <% if @poll.about %>
     <p>
       <%= t(
-        'notifier.pending_poll_email.affected_user_html',
-        user: @poll.affected_user.full_name
+        'notifier.pending_poll_email.about_for_html',
```

Comment:
¿Porque ```about_for```? Suena raro, ```about``` solo creo que quedaría mejor.

Reply:
porque el `about` solo es sin parámetro, por eso le puse un for :)

Reply:
Pero ```about_html``` =)